react-select/react-select.d.ts

```diff
@@ -1,80 +1,221 @@
-// Type definitions for react-select v0.9.10
+// Type definitions for react-select v1.0.0
 // Project: https://github.com/JedWatson/react-select
-// Definitions by: ESQUIBET Hugo <https://github.com/Hesquibet/>
```

Comment:
please keep original author name. he is first user of this definitions. add your name below.